package.json

```diff
@@ -51,7 +51,7 @@
     "deps:install": "npm ci",
     "deps:truncate": "npm prune --production",
     "typecheck": "tsc -p . --noEmit",
-    "test": "exit 0",
+    "test": "npx playwright test",
```

Comment:
It is necessary to add a section about tests in the readme, how to run them and how to update them

Reply:
Added more info about testing in README: https://github.com/diplodoc-platform/components/pull/327/commits/fb2595c97bff2fd0851ce55c40c6468ea951f3e8